wp-content/plugins/zippicks-favorites/includes/API/RestController.php

```
@@ -0,0 +1,453 @@
+        }
+    }
+    
+    /**
+     * Delete favorite
+     */
+    public function delete_favorite($request) {
+        $user_id = get_current_user_id();
+        $favorite_id = $request->get_param('id');
+        
+        try {
+            $response = $this->api_client->remove_favorite($user_id, $favorite_id);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$response'. (Unused Code Rules)

(UnusedLocalVariable)